src/textual/widgets/_select.py

```diff
@@ -278,6 +278,44 @@ def __init__(
         self._value: SelectType | None = value
         self._options = options
 
+    @classmethod
+    def from_values(
+        cls,
+        opts: [SelectType],
+        *,
+        prompt: str = "Select",
+        allow_blank: bool = True,
+        value: SelectType | None = None,
+        name: str | None = None,
+        id: str | None = None,
+        classes: str | None = None,
+        disabled: bool = False,
+    ):
+        """Initialize the Select control with options specified by an iterator
+
+        Args:
+         options: Options to select from.
+         prompt: Text to show in the control when no option is select.
+         allow_blank: Allow the selection of a blank option.
+         value: Initial value (should be one of the values in `options`).
+         name: The name of the select control.
+         id: The ID of the control the DOM.
+         classes: The CSS classes of the control.
+         disabled: Whether the control is disabled or not.
+        """
+        opts_iterator = [(str(o), o) for o in opts]
+
+        return cls(
+            opts_iterator,
```

Comment:
We avoid shortening names as much as possible, so feel free to type out `options_iterator`, `options`, and `option`, or rename to something else if you prefer.